gips/0001-gip-process.md

```diff
@@ -35,80 +35,138 @@ The high-level process is captured in the diagram below.
+A template has been added to this repo called "0000-template.md" which provides additional detail on these sections and should be kept up to date with the latest process as described in this proposal.
 
 # Editors
 
-Editors are the maintainers of this repo and the managers of this process. As the name implies, their role is purely editorial, and they should advance proposals to stages on the basis of how well they have been constructed per the guidance in this proposal, rather than based on the likelihood for the proposal to be included in the protocol.
+Editors are the maintainers of this repo and the managers of this process. As the name implies, their role is purely editorial, and they should advance proposals to stages on the basis of whether specific criteria have been met, rather than based on the likelihood for the proposal to be included in the protocol.
 
 Editors do not play a role in decentralized governance, but rather lend their expertise to help proposals get written that are clear, concise, and advocate effectively for their respective ideas.
 
+The Editors' role is not to act as gatekeepers, so proposals can advance up to being accepted by the Council even in case of Editor inactivity.
+
 The current GIP editors are:
 
-- Ariel Barmat
-- Brandon Ramirez
-- Jannis Pohlmann
+- Michael Macaulay
+- Pedro Diogo
 
-Over time, as more community members build deep familiarity with the protocol design and associated software, more editors will be added to this list.
+Over time, as more community members build deep familiarity with the protocol design and associated software, more editors can be added to this list.
 
 # Submitting a Proposal
 
-The GIPs repository is hosted on [Radicle](https://radicle.xyz/), a crypto-native decentralized alternative to Github. Currently, Radicle does not support an issue or Pull Request workflow, although this is being worked on.
+The GIPs repository is [hosted on GitHub](https://github.com/graphprotocol/graph-improvement-proposals).
+
+Proposals can be submitted by forking the repository and creating a Pull Request to include the proposal in Markdown format, following the GIP-0000 template.
+
+Discussions around proposals should be directed to the [GIPs and Governance](https://forum.thegraph.com/c/governance-gips/) category of The Graph's forum.
+
+**To avoid conflicts in GIP numbers, proposals should be posted as a Pull Request on GitHub with a GIP number _before_ being posted to the forum**.
+
+After creating the Pull Request, post a thread in the GIPs and Governance category with a link to the Pull Request. The post should have the same title as the GIP, and include the Abstract and Motivation sections but linking to GitHub for the full GIP text. (Older GIPs may include the full proposal text in the forum post, but this is now discouraged as it is hard to keep the forum and GitHub versions in sync).
```

Comment:
@pcarranzav with the new [GIP Tracker](https://www.notion.so/thegraphfoundation/7262cfb77f544a979caab17c4d4bbdc5?v=f10e2221e45d4a3c89e37b89d8d0bc46&pvs=4), we're considering having a  GPT-powered TLDR/Summary for all GIPs, so people can quickly understand what the whole proposal is all about without having to read the entire thing (after clicking a link from the forum). It might make sense for us to do the same in the Forum discussion for all subsequent GIPs, with some automation. Do you see any problem with this?

Reply:
I think that'd be pretty cool! I would just suggest adding a disclaimer noting this is a GPT-powered summary so it might be biased